### ISSUE TITLE
Documentation and README

### ISSUE THREAD

**Issue by alxhoff** [alxhoff/STM32-uGFX-Cmake#11]
Documentation of how to use the build should be here and in the pdf.

I would say basic usage, ie. building and flashing, can go in the readme while details VS code integration should go in the wiki.

Things to be documented:
- Basics of cmake/make usages
  - running cmake
  - make targets
- Eclipse (deprecated)
- VS Code

**Comment by PhilippvK** [comment]
I already planned to do it as you described. Just focussing on the other stuff (Emulator) until before next semester.

**Comment by alxhoff** [comment]
Yep no worries, I wouldn't invest much/any time into this until after this semester. More of a reminder for us for later. I'm not really sure how to test the VS code PR as I have no idea how to use VS code :P did you also do/are doing VS code stuff for the emulator?

**Comment by PhilippvK** [comment]
I would suggest to wait until I have written a short documentation for the vscode stuff.

I am currently finalizing the vscode config for the emulator. You can try it out when i have uploaded the latest virtualbox image later!

**Comment by alxhoff** [comment]
ok awesome. "looking forward" to using an IDE.....
